manifests/init.pp

```diff
@@ -170,6 +173,7 @@
     ssl_cert_name          => 'broker',
     ssl_cert_db            => $certs::nss_db_dir,
     ssl_cert_password_file => $certs::qpid::nss_db_password_file,
+    wcache_page_size       => $::katello_devel::params::qpid_wcache_page_size
```

Comment:
Don't reference the params here, otherwise you will never use any value passed by the user.